src/SME.SGP.Aplicacao/Queries/Turma/ListagemTurmasComComponente/ObterTurmasComComponentesQueryHandler.cs

```diff
@@ -24,7 +24,7 @@ public async Task<PaginacaoResultadoDto<RetornoConsultaListagemTurmaComponenteDt
             var turmaCodigo = request.TurmaCodigo == null ? 0 : long.Parse(request.TurmaCodigo);
 
             var httpClient = httpClientFactory.CreateClient("servicoEOL");
-            var resposta = await httpClient.GetAsync($"turmas/ues/{request.UeCodigo}/modalidades/{(int)request.Modalidade}/anos/{request.AnoLetivo}/componentes?codigoTurma={turmaCodigo}&ehProfessor={request.EhProfessor}&codigoRf={request.CodigoRf}&qtdeRegistros={request.QtdeRegistros}&qtdeRegistrosIgnorados={request.QtdeRegistrosIgnorados}");
+            var resposta = await httpClient.GetAsync($"turmas/ues/{request.UeCodigo}/modalidades/{(int)request.Modalidade}/anos/{request.AnoLetivo}/componentes?codigoTurma={turmaCodigo}&ehProfessor={request.EhProfessor}&codigoRf={request.CodigoRf}&qtdeRegistros={request.QtdeRegistros}&qtdeRegistrosIgnorados={request.QtdeRegistrosIgnorados}&consideraHistorico={request.ConsideraHistorico}");
 
             if (resposta.IsSuccessStatusCode)
             {
```

Comment:
Tem algum perigo de um desses parâmetros ir nulo ou vazio, pq isso vai quebrar a montagem, talvez nesse caso teria que ser montado dinamicamente.

Reply:
Mas se ele não for passado ele será um default = false